src/github3/pulls.py

```diff
@@ -477,7 +477,8 @@ def issue_comments(self, number=-1, etag=None):
         return self._iter(int(number), url, IssueComment, etag=etag)
 
     @requires_auth
-    def merge(self, commit_message=None, sha=None, merge_method='merge'):
+    def merge(self, commit_message=None, sha=None, merge_method='merge',
+              commit_title=None):
```

Comment:
Please add another paragraph to the docstring with

```
.. versionchanged:: 1.3.0

    The ``commit_title`` parameter has been added to allow users to set the merge commit title.

```

(With proper line-wrapping, etc.)

Reply:
Thanks, it is fixed now